src/main/java/core/basesyntax/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.FruitStorage;
+
+public class BalanceOperationHandler implements OperationHandler {
```

Comment:
According to the SOLID principles and the task checklist, this class should not depend directly on a concrete implementation of `FruitStorage`. Instead, consider depending on an abstraction (e.g., an interface) to make the code more flexible and easier to test.